src/Interactive/CsiCore/Csi.cs

```diff
@@ -17,14 +15,18 @@ internal static int Main(string[] args)
         {
             try
             {
-                var responseFile = Path.Combine(AppContext.BaseDirectory, InteractiveResponseFileName);
+                // Note that AppContext.BaseDirectory isn't necessarily the directory containing csi.exe.
+                // For example, when executed via corerun it's the directory containing corerun.
+                string csiDirectory = Path.GetDirectoryName(typeof(Csi).GetTypeInfo().Assembly.ManifestModule.FullyQualifiedName);
```

Comment:
I'm not opposed to the change, but that seems like a bug in corerun. Is there a better example to use in the comment than an internal coreclr host?